Interfaces/API/Dirac.py

```diff
@@ -2600,7 +2600,9 @@ def __getJDLParameters( self, jdl ):
     :param jdl: a JDL
     :type jdl: ~DIRAC.Interfaces.API.Job.Job or str or file
     """
-    if isinstance(jdl, DIRAC.Interfaces.API.Job.Job):
+    # import here, becase Job module imports this module
+    from DIRAC.Interfaces.API.Job import Job
```

Comment:
Maybe better to use hasattr to check if jdl has `_toJDL` instead of importing+isinstance here?